test/suite/test_rollback_to_stable19.py

```diff
@@ -102,11 +102,17 @@ def test_rollback_to_stable_no_history(self):
         # Search for the key so we position our cursor on the page that we want to evict.
         self.session.begin_transaction("ignore_prepare = true")
         evict_cursor.set_key(1)
-        evict_cursor.search()
+        self.assertEquals(evict_cursor.search(), WT_NOTFOUND)
         evict_cursor.reset()
         evict_cursor.close()
         self.session.commit_transaction()
 
+        self.session.begin_transaction("ignore_prepare = true")
```

Comment:
As you are already testing the on-disk visibility separately in the above test, this test change may not be needed. If you want to keep this change also, then add a similar case in the below flow also `test_rollback_to_stable_with_history`.

Reply:
OK